src/app/services/ccd-config/ccd-case.config.ts

```diff
@@ -13,10 +17,21 @@ import {AppConfigService} from '../config/configuration.services';
 export class AppConfig extends AbstractAppConfig {
 
   protected config: CaseEditorConfig;
+  public workallocationUrl: string;
 
-  constructor(private appConfigService: AppConfigService) {
+  constructor(
+    private appConfigService: AppConfigService,
+    private readonly featureToggleService: FeatureToggleService
+    ) {
     super();
     this.config = this.appConfigService.getEditorConfiguration() || {};
+    this.featureToggleWorkAllocation();
+  }
+
+  private featureToggleWorkAllocation(): void {
+    this.featureToggleService.isEnabled(AppConstants.FEATURE_NAMES.workAllocation).subscribe(
+      isFeatureEnabled => this.workallocationUrl = AppUtils.getFeatureToggledUrl(isFeatureEnabled, WorkAllocationTaskService.WorkAllocationUrl)
+      );
```

Comment:
Something odd going on with the indenting here...

Reply:
Used Prettier now. fixed it